content/en/error_tracking/backend/getting_started.md

```diff
@@ -0,0 +1,439 @@
+
+4. Run the following command to deploy the agent:
+   ```shell
+   helm install datadog-agent -f datadog-values.yaml datadog/datadog
+   ```
+5. After waiting a few minutes for the Datadog Cluster Agent changes to apply, restart your applications.
+
+{{< /collapse-content >}}
+
+[7]: https://v3.helm.sh/docs/intro/install/
+[8]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+[9]: https://app.datadoghq.com/organization-settings/api-keys
+[10]: /getting_started/site/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Using Datadog tracing libraries
+
+To automatically instrument your application with Datadog libraries:
+
+1. [Install and configure the Agent](#install-and-configure-the-agent).
+2. [Add the Datadog tracing library to your code](#instrument-your-application).
+
+#### Install and configure the Agent
+
+Install the Datadog Agent by following the [relevant documentation][11].
+
+To configure the agent for Error Tracking Backend only, you must be running Agent v7.61+.
+
+{{< tabs >}}
+{{% tab "Linux host or VM" %}}
+
+1. Open the [datadog.yaml configuration file][12].
+2. Add `core_agent` and `apm_config` as top-level attributes anywhere in the configuration file with the following settings:
+
+   ```yaml
+   core_agent:
+     enabled: false
+   apm_config:
+     error_tracking_standalone:
+       enabled: true
+   ```
+
+3. [Restart the Agent][13].
+
+[12]: /agent/configuration/agent-configuration-files/
+[13]: /agent/configuration/agent-commands/#restart-the-agent
+
+{{% /tab %}}
+{{% tab "Docker" %}}
+
+If you're using the Docker containerized Agent, set the following environment variables:
+- `DD_CORE_AGENT_ENABLED=false`
+- `DD_APM_ERROR_TRACKING_STANDALONE_ENABLED=true`
+
+Here's an example of how you can include these settings in your Docker run command:
+
+```shell
+docker run -d --name datadog-agent \
+           --cgroupns host \
+           --pid host \
+           -e DD_API_KEY=<DATADOG_API_KEY> \
+           -e DD_CORE_AGENT_ENABLED=false \
+           -e DD_APM_ERROR_TRACKING_STANDALONE_ENABLED=true \
+           -v /var/run/docker.sock:/var/run/docker.sock:ro \
+           -v /proc/:/host/proc/:ro \
+           -v /opt/datadog-agent/run:/opt/datadog-agent/run:rw \
+           -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+           gcr.io/datadoghq/agent:latest
+```
+
+{{% /tab %}}
+{{% tab "Kubernetes" %}}
+
+If you're deploying the Agent in Kubernetes, make the following changes in your Helm chart in addition to your Agent configuration:
+
+```yaml
+agents:
+  containers:
+    agent:
+      env:
+        - name: DD_CORE_AGENT_ENABLED
+          value: "false"
+datadog:
+[...]
+  processAgent:
+    enabled: false
+    containerCollection: false
+[...]
+  apm:
+    errorTrackingStandalone:
+      enabled: true
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Instrument your application
+
+Follow the relevant [documentation][14] to set up your application to send traces using one of the official Datadog tracing libraries.
+
```

Comment:
We probably want to add a doc page / link to a page about manually sending errors (through span events)